cmd/opm/serve/serve.go

```diff
@@ -107,5 +136,18 @@ func (s *serve) run(ctx context.Context) error {
 		return grpcServer.Serve(lis)
 	}, func() {
 		grpcServer.GracefulStop()
+
+		if s.memProfile != "" {
+			f, err := os.Create(s.memProfile)
+			if err != nil {
+				s.logger.Fatalf("could not create memory profile: %v", err)
+			}
+			defer f.Close()
+			runtime.GC()
+			if err := pprof.WriteHeapProfile(f); err != nil {
```

Comment:
Might be helpful to write a log here that the heap profile has been written to f

Reply:
Done